openreview/conference/templates/programchairWebfield.js

```diff
@@ -1890,7 +1890,6 @@ var buildPaperTableRow = function(note, reviewerIds, completedReviews, metaRevie
   var cellCheck = { selected: false, noteId: note.id };
 
   // Build Note Summary Cell
-  note.content.authors = null;  // Don't display 'Blinded Authors'
```

Comment:
With this line removed do all the cells of the Paper Status table display "Blinded Authors"?

Reply:
Because I need the authors names to put them in the export file. If I keep this line, single-blind conferences can not export the author names. May be I can do this only for double-blind conferences.

Reply:
Right I understand that, I'm talking about the Paper Summary column in the paper status table. Maybe we can modify the templates to not include "Blinded Authors" in cell. Honestly though it's not a big deal and we can change this another time.